bigbluebutton-html5/imports/utils/safari-webrtc.js

```diff
@@ -0,0 +1,65 @@
+const iceServersList = [
```

Comment:
I think you should use the `fetchStunTurnServers`, if possible, to make it consistent with the actual audio/video negotiation procedures. You could use this list as a fallback if `fetchStunTurnServers` returns nothing (that's what is done with video/screensharing).

Reply:
It was changed to use fetchStunTurnServers